app/models/practice.rb

```diff
@@ -1,10 +1,12 @@
 class Practice < ApplicationRecord
   acts_as_list
   # has_attached_file :main_display_image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
-  has_attached_file :main_display_image, default_url: "/images/:style/missing.png"
-  has_attached_file :origin_picture, default_url: "/images/:style/missing.png"
+  has_attached_file :main_display_image
```

Comment:
we're just putting a color for the background. this throws an error since we don't have that placeholder image